# Table of contents

* [VCPA](README.md)
* [Introduction](introduction.md)
* [Step 1: Set up the Amazon Web Services \(AWS\) environment](step-1-set-up-the-amazon-web-services-aws-environment/README.md)
  * [1.1 Create AWS account](step-1-set-up-the-amazon-web-services-aws-environment/1.1-create-aws-account.md)
  * [1.2 Configure your computing environment and login to AWS](step-1-set-up-the-amazon-web-services-aws-environment/1.2-configure-your-computing-environment-and-login-to-aws.md)
  * [1.3 Setup a S3 bucket \(simple storage solution for AWS\) for hosting sequencing data](step-1-set-up-the-amazon-web-services-aws-environment/1.3-setup-a-s3-bucket-simple-storage-solution-for-aws-for-hosting-sequencing-data.md)
  * [1.4 Install AWS command line software for accessing S3 bucket via command line interface](step-1-set-up-the-amazon-web-services-aws-environment/1.4-install-aws-command-line-software-for-accessing-s3-bucket-via-command-line-interface.md)
  * [1.5 Install StarCluster for AWS instance provisioning \(optional\)](step-1-set-up-the-amazon-web-services-aws-environment/1.5-install-starcluster-for-aws-instance-provisioning-optional.md)
* [Step 2: Create your tracking database instance](step-2-create-your-tracking-database-instance/README.md)
  * [Option 1: Setup sample tracking database using Public AMI \(recommended\)](step-2-create-your-tracking-database-instance/option-1-setup-sample-tracking-database-using-public-ami-recommended.md)
  * [Option 2: Setup sample tracking database using Docker](step-2-create-your-tracking-database-instance/option-2-setup-sample-tracking-database-using-docker.md)
* [Step 3: Configure your project information in the tracking database](step-3-configure-your-project-information-in-the-tracking-database/README.md)
  * [3.1 List all projects in the tracking database](step-3-configure-your-project-information-in-the-tracking-database/3.1-list-all-projects-in-the-tracking-database.md)
  * [3.2 Create the project in the tracking database](step-3-configure-your-project-information-in-the-tracking-database/3.2-create-the-project-in-the-tracking-database.md)
* [Step 4: Upload sequencing data to your S3 bucket](step-4-upload-sequencing-data-to-your-s3-bucket.md)
* [Step 5: Configure your samples information in the tracking database](step-5-configure-your-samples-information-in-the-tracking-database/README.md)
  * [5.1 Input the sample information to the tracking database](step-5-configure-your-samples-information-in-the-tracking-database/5.1-input-the-sample-information-to-the-tracking-database.md)
  * [5.2 Populate the tracking database with the S3 paths for the samples to be processed](step-5-configure-your-samples-information-in-the-tracking-database/5.2-populate-the-tracking-database-with-the-s3-paths-for-the-samples-to-be-processed.md)
  * [5.3 Populate the tracking database with the designated result folder for each sample to be processed](step-5-configure-your-samples-information-in-the-tracking-database/5.3-populate-the-tracking-database-with-the-designated-result-folder-for-each-sample-to-be-processed.md)
  * [5.4 Input PCR protocol information into the tracking database](step-5-configure-your-samples-information-in-the-tracking-database/5.4-input-pcr-protocol-information-into-the-tracking-database.md)
  * [5.5 Add the capture kit information \(WES sample only\) into the tracking database](step-5-configure-your-samples-information-in-the-tracking-database/5.5-add-the-capture-kit-information-wes-sample-only-into-the-tracking-database.md)
  * [5.6 Generate an ID to represent the capture kit information \(WES sample only\)](step-5-configure-your-samples-information-in-the-tracking-database/5.6-generate-an-id-to-represent-the-capture-kit-information-wes-sample-only.md)
* [Step 6: Submit a job to process one whole genome \(WGS\) / whole exome \(WES\) sample](step-6-submit-a-job-to-process-one-whole-genome-wgs-whole-exome-wes-sample/README.md)
  * [6.1 Update vcpa-pipeline bitbucket contents](step-6-submit-a-job-to-process-one-whole-genome-wgs-whole-exome-wes-sample/6.1-update-vcpa-pipeline-bitbucket-contents.md)
  * [6.2 Choose which workflow to use](step-6-submit-a-job-to-process-one-whole-genome-wgs-whole-exome-wes-sample/6.2-choose-which-workflow-to-use.md)
  * [6.3 Enter your AWS credentials into the workflow script](step-6-submit-a-job-to-process-one-whole-genome-wgs-whole-exome-wes-sample/6.3-enter-your-aws-credentials-into-the-workflow-script.md)
  * [6.4 Launch Amazon EC2 Spot Instances via starcluster](step-6-submit-a-job-to-process-one-whole-genome-wgs-whole-exome-wes-sample/6.4-launch-amazon-ec2-spot-instances-via-starcluster.md)
* [Step 7: Review quality metrics of processed data](step-7-review-quality-metrics-of-processed-data.md)
* [Step 8: Generating Project-level VCF via joint genotyping](step-8-generating-project-level-vcf-via-joint-genotyping.md)
* [Optional: Change software versions and dependencies of the VCPA workflow](optional-change-software-versions-and-dependencies-of-the-vcpa-workflow.md)

